plugins/parsers/prometheusremotewrite/parser_test.go

```diff
@@ -40,7 +40,30 @@ func TestParse(t *testing.T) {
 	inoutBytes, err := prompbInput.Marshal()
 	require.NoError(t, err)
 
-	expected := []telegraf.Metric{
+	expectedV1 := []telegraf.Metric{
+		testutil.MustMetric(
```

Comment:
Please use `metric.New`. We should remove the testutil wrapper sometimes in the future I think... Same for other occurrences below.